README.md

```diff
@@ -68,6 +67,18 @@ workers implement the Remote Worker API.
 * [Scoot](https://github.com/twitter/scoot) (open source)
 * [Turbo Cache](https://github.com/allada/turbo-cache) (open source)
 
+### Workers
+Servers are responsible for managing a pool of workers.
```

Comment:
I don't think I'd phrase it in this way. Not all server implementations "manage" the worker pools, and it would be a legitimate use of the API to multiplex work from many clients onto a single, powerful/rare machine (and thus not a pool, per se). Maybe "Servers generally distribute work to a fleet of workers. While there is no standard API for communication between the server and workers, links to the APIs from some existing implementations are provided as a reference below. Adhering to any one of these protocols is not a requirement."